scratch/copy-mk.py

```diff
@@ -166,13 +173,18 @@ async def async_main(args) -> int:
         "delay_centre_frequency",
         "subarray_product_id",
     ]
-    cbf_sensor_values = await multi_sensor_values(
-        portal_client,
-        [f"{prefix}_{name}" for name in cbf_sensor_names],
-    )
+    sensor_names = [f"{prefix}_{name}" for name in cbf_sensor_names]
+    if args.sdp is not None:
+        sdp_prefix = await portal_client.sensor_subarray_lookup("sdp", None)
+        sdp_subarray_product_id = await single_sensor_value(portal_client, f"{sdp_prefix}_subarray_product_ids")
```

Comment:
`_subarray_product_ids` plural?

Reply:
Yes - I have no idea why CAM named the sensor like this.